sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -283,10 +294,15 @@ class SessionCatalog(
    * explicitly specified.
    */
   def isTemporaryTable(name: TableIdentifier): Boolean = {
-    !name.database.isDefined && tempTables.contains(formatTableName(name.table))
+    name.database.isEmpty && tempTables.contains(formatTableName(name.table))
   }
 
   /**
+   * Return whether View is supported
+   */
+  def isViewSupported: Boolean = false
```

Comment:
OK. I see. We cannot create a view in SQLContext because the parser rule is in HiveSqlParser, right? So, it is better just remove this method and if a user calls drop view in SQLContext, because the view is not defined, they can still see an error message, right?